src/server.js

```diff
@@ -0,0 +1,38 @@
+const Hapi = require("hapi")
```

Comment:
Yeah, I've been messing around with it in some of my free time since you introduced me to it. Big fan.